### PR TITLE
chore: adding coverage test for custom batch properties

### PR DESCRIPTION
**Pull request by tool3**
[trello](https://trello.com/c/e2MbEs0r)  
adding test for custom batch properties

### CODE REVIEW

File: coverage-tests.js [hunk 1]
```diff
@@ -1314,6 +1314,21 @@ test('should send dom of version 11', {
 
 // #region OTHERS
 
+test('should send custom batch properties', {
+  page: 'Default',
+  config: {
+    properties: {
```
Comment (amitzur) [inline coverage-tests.js:1320]: @tool3 it should be this structure:
```
config: {
  batch: {
    properties: [{name: 'custom prop', value: 'custom value'}]
  }
}
```

Comment (tool3) [reply coverage-tests.js:1320]: thanks ! makes sense !

File: coverage-tests.js [hunk 2]
```diff
@@ -1314,6 +1314,23 @@ test('should send dom of version 11', {
...
+      properties: {
+        custom_prop: 'custom value'
+      }
+    }
+  },
+  test({eyes, assert, helpers}) {
+    eyes.open({appName: 'Eyes Selenium SDK - Custom Batch Properties', viewportSize});
+    const result = eyes.close();
+    const info = helpers.getBatchInfo(result); 
+    assert.equal(info.properties, {custom_prop: 'custom_prop'})
```
Comment (amitzur) [inline coverage-tests.js:1330]: Also here (I think we don't support arrays in `assert.equal`):
```
assert.equal(info.properties.length, 1)
assert.equal(info.properties[0], {name: 'custom prop', value: 'custom value'})
```

**Comment by amitzur** [comment]
@tool3 great!
Please add `skipEmit` in the overrides of other languages

**Pull request closed without merging**